django/chat/templates/chat/modals/presets/presets_form.html

```diff
@@ -124,7 +128,7 @@
 </div>
 
 <script>
-  document.body.addEventListener('htmx:afterSettle', function() {
+  document.querySelector('#presets-modal').addEventListener('htmx:afterSettle', function() {
```

Comment:
All this code was being run anytime **anything** was swapped into the page. Limiting the listener only to the preset modal solves this problem.